docs/miscellaneous/logging.rst

```diff
@@ -0,0 +1,29 @@
+.. _logging_section:
+
+Logging
+=======
+
+In **alchemlyb**, we use :mod:`loguru` for logging. By default, the
```

Comment:
add the appropriate intersphinx links to conf.py to make links to loguru docs work